### PR TITLE
Updates DE Translation: tags -> labels

### PR DESCRIPTION
**Pull request by rom-1**
in addition to PR3446. see https://github.com/FreshRSS/FreshRSS/pull/3446

Edit: There is no real translation for 'labels' in german, as this word is 1:1 used in our language for technical 'labels'. btw google does the same with gmail: they are simply using the word "labels" in german.

### CODE REVIEW

File: app/i18n/de/sub.php [hunk 1]
```diff
@@ -124,10 +124,10 @@
 		'add' => 'Feed oder Kategorie hinzufügen',
 		'add_category' => 'Kategorie hinzufügen',
 		'add_feed' => 'Feed hinzufügen',
-		'add_tag' => 'Tag hinzufügen',
-		'delete_tag' => 'Tag löschen',
+		'add_label' => 'Label hinufügen',
```
Comment (aledeg) [inline app/i18n/de/sub.php:127]: It seems that there is a `z` missing?

File: app/i18n/de/index.php [hunk 2]
```diff
@@ -53,12 +53,12 @@
...
 		'stats' => 'Statistiken',
 		'subscription' => 'Abonnementverwaltung',
-		'tags' => 'Meine Tags',
+		'tags' => 'Meine Labels',
 		'unread' => 'Nur ungelesene zeigen',
 	),
 	'share' => 'Teilen',
 	'tag' => array(
-		'related' => 'Verwandte Tags',
+		'related' => 'Artikel-Labels',
```
Comment (aledeg) [inline app/i18n/de/index.php:61]: This is actually an article tag not a label :)

File: app/i18n/de/sub.php (hunk 1 above)
Comment (rom-1) [reply app/i18n/de/sub.php:127]: correct, sorry, "hinzufügen"

File: app/i18n/de/conf.php [hunk 3]
```diff
@@ -28,7 +28,7 @@
 			'display_authors' => 'Autoren',
 			'entry' => 'Artikel-Symbole',
 			'publication_date' => 'Datum der Veröffentlichung',
-			'related_tags' => 'Verwandte Tags',
+			'related_tags' => 'Artikel-Labels',
```
Comment (aledeg) [inline app/i18n/de/conf.php:31]: article tag as well

File: app/i18n/de/index.php (hunk 2 above)
Comment (rom-1) [reply app/i18n/de/index.php:61]: whoops, sorry :)

File: app/i18n/de/sub.php (hunk 1 above)
Comment (aledeg) [reply app/i18n/de/sub.php:127]: Perfect!

File: app/i18n/de/conf.php (hunk 3 above)
Comment (rom-1) [reply app/i18n/de/conf.php:31]: are you sure about this? I am refering to Settings Display -> in german, this looks like this right now:
2021-02-14_15h46_24

So if I check this for "bottom line", it shows "my labels" in the bottom line. therefore, this seem to relate to labels?

**Comment by aledeg** [comment]
@rom-1 I've updated my PR with your translations. Please review!
If it's correct, I think you can close this one.

File: app/i18n/de/conf.php (hunk 3 above)
Comment (Frenzie) [reply app/i18n/de/conf.php:31]: Labels are added by the user and tags come from the feed. Are they both toggled by the same setting?

Comment (rom-1) [reply app/i18n/de/conf.php:31]: Update: ok, I don't get it. Even if I "uncheck" the "verwandte tags" (related tags) option in the bottom line (fußzeile), the "my labels" is showing in the bottom line in the main view. so this setting does not seem to have any effect at all.

Comment (aledeg) [reply app/i18n/de/conf.php:31]: This toggle show article tags (from feed) in the article itself.
Labels are user generated. You cannot turn off labels. But you can turn off tags.

Comment (Frenzie) [reply app/i18n/de/conf.php:31]: No, that's what I was (more or less) expecting to happen. That means you can't turn them off in the display there, although one might wonder why not.

For tags you need an article that actually has some. They come from the feed.
Screenshot_2021-02-14_15-53-15

Comment (rom-1) [reply app/i18n/de/conf.php:31]: oh. ok, then I mixed it up. I was not aware that you cannot turn of labels, sorry. 
seems I simply do not have any feeds which support article tags then.

Comment (aledeg) [reply app/i18n/de/conf.php:31]: You explain it better than I did :)
I was trying to say the same thing but in the worst possible way

**Comment by rom-1** [comment]
Changes have been implemented to #3446

**Pull request closed without merging**